libs/detectors/x86/detector.py

```diff
@@ -1,3 +1,37 @@
+import json
+import os
+from pathlib import Path
+
+
+def get_or_create_model_json_file(config, device, camera_id):
+    """
+    If no model directory was created for a given device, the default values will be taken from the configuration file
+    and a model_<device>.json file will be created.
+    """
+    base_path = os.path.join(config.get_section_dict("App")["EntityConfigDirectory"], "sources", camera_id)
+    models_directory_path = os.path.join(base_path, "ml_models")
+    json_file_path = os.path.join(models_directory_path, f"model_{device}.json")
+
+    if not os.path.exists(json_file_path):
+        # Hypothesis: source config directory (base_path) should always exists.
+        if not os.path.exists(models_directory_path):
+            Path(models_directory_path).mkdir(parents=True, exist_ok=True)
+
+        # Create .json file
+        json_content = {
+            "model_name": config.get_section_dict("Detector")["Name"],
+            "variables": {
+                key: value for key, value in config.get_section_dict("Detector").items() if key not in ["Name", "Device"]
+            }
+        }
+        with open(json_file_path, 'x+') as outfile:
+            json.dump(json_content, outfile)
+        os.chmod(json_file_path, 0o777)
```

Comment:
Why is this? Because access is only alowed to the docker user?

Reply:
You definitely don't need execute permissions on a json file

Reply:
rw access for everyone maybe?

Reply:
Great!